dom0-updates/qubes-dom0-updates.cron

```diff
@@ -4,6 +4,8 @@ if [ "$(qvm-features dom0 service.qubes-update-check || echo 1)" != 1 ]; then
     exit 0
 fi
 
+qvm-features dom0 last-updates-check "$(date +'%Y-%m-%d %T')"
+
 # If no updates available - exit here
 qubes-dom0-update --check-only >/dev/null && exit
 RETCODE=$?
```

Comment:
If I'm not mistaken then this sets the date even if the update check failed?

Reply:
I think you are correct. @piotrbartman can you look at it ?